tests/pytorch/test_numerics.py

```diff
@@ -1116,32 +1119,51 @@ def test_dpa_accuracy(dtype, bs, model):
         assert_allclose(te_output, torch_output, atol=5e-2, rtol=1e-2)
 
 
+class TestReturnBiasModule(nn.Module):
+    def __init__(self, mod, **kwargs):
+        super().__init__()
+        self.te_module = mod(**kwargs)
+        self.return_bias = kwargs["return_bias"]
+        self.bias = kwargs["bias"]
+
+    def forward(self, x):
+        if self.bias and self.return_bias:
+            out, bias = self.te_module(x)
+            return out + bias
+        return self.te_module(x)
```

Comment:
Do we handle the case where `bias=False` and `return_bias=True`?
```suggestion
        if self.return_bias:
            out, bias = self.te_module(x)
            if self.bias:
                out = out + bias
            return out
        return self.te_module(x)
```